app/core/component/course-list/course-list.template.html

```diff
@@ -1,36 +1,55 @@
 <div class="course-container"
      ng-show="$ctrl.coursesIsLoaded">
-    <ul class="course-list"
-        ng-repeat="course in $ctrl.courses |
-        limitTo:$ctrl.coursePullSize |
-        filter: $ctrl.filterValue">
-        <li>
-            <div class="info">
-                <h3 class="flex-justify-between">
-                    {{course.title}}
-                    <span>{{course.displayDuration}}&nbsp{{course.displayDate}}</span>
-                </h3>
-                <p>{{course.description}}</p>
-            </div>
-            <div class="action-bar">
-                <button type="button"
-                        class="btn btn-edit action-btn"
-                        ng-click="$ctrl.pushCourseToEditForm(course)">
-                    <span>Edit</span>
-                </button>
-                <button type="button"
-                        class="btn btn-delete action-btn"
-                        ng-click="$ctrl.deleteCourse(course)">
-                    <span>Delete</span>
-                </button>
+    <ul class="course-list">
+        <li class="course-item"
+            state-by-date="course"
+            ng-repeat="course in $ctrl.courses |
+                limitTo:$ctrl.coursePullSize |
+                filter: $ctrl.filterValue |
+                orderBy: $ctrl.creationDateToTime">
+            <div class="course-wrapper"
+                 ng-class="$ctrl.topRatedClassIsIt(course)">
```

Comment:
isTopRated will be more readable

Reply:
renamed
ok but it return class name, it's okay for 'is...' function return not booleam?

Reply:
According to your function implementation (if or nothing in return), I'd say, that it is more like boolean.